### PR TITLE
[DPUB-7652] use isHotkey for non-Latin keyboard support

### PR DESCRIPTION
**Pull request by Ben-Wormald**

### CODE REVIEW

File: packages/@bbc/old-optimo-slate-react/package.json [hunk 1]
```diff
@@ -1,7 +1,10 @@
 {
-  "name": "slate-react",
+  "name": "@bbc/old-optimo-slate-react",
   "description": "A set of React components for building completely customizable rich-text editors.",
-  "version": "0.21.0",
+  "version": "0.21.2",
```
Comment (tamsingreen) [inline packages/@bbc/old-optimo-slate-react/package.json:4]: 0.21.1?

**Review by tamsingreen — APPROVED** [review]

File: packages/@bbc/old-optimo-slate-react/package.json (hunk 1 above)
Comment (Ben-Wormald) [reply packages/@bbc/old-optimo-slate-react/package.json:4]: had a permissions issue with 0.21.1 so we had to publish again

Comment (Ben-Wormald) [reply packages/@bbc/old-optimo-slate-react/package.json:4]: (well we thought we did at the time, it did actually work!)

Comment (tamsingreen) [reply packages/@bbc/old-optimo-slate-react/package.json:4]: Ah - need to update the other PR to use 0.21.2 then?

Comment (Ben-Wormald) [reply packages/@bbc/old-optimo-slate-react/package.json:4]: I think the two versions should be the same

**Pull request merged**